src/status_im/utils/utils.cljs

```diff
@@ -84,14 +88,21 @@
       (str/trim)))
 
 (defn first-index
-  [cond coll]
```

Comment:
it would be better to name this param pred because cond is a clojure.core function and predicate is the idiomatic clojure term for a condition